src/segments/owm.go

```diff
@@ -88,14 +90,21 @@ func (d *Owm) getResult() (*owmDataResponse, error) {
 		apikey = d.env.Getenv(PoshOWMAPIKey)
 	}
 
-	location := d.props.GetString(Location, "De Bilt,NL")
+	if len(apikey) == 0 {
+		return nil, errors.New("no api key found")
+	}
 
-	location = url.QueryEscape(location)
+	location := d.props.GetString(Location, "De Bilt,NL")
+	if len(location) == 0 {
```

Comment:
This will never be true, right? We default to a value when fetching the `location` from the `properties` which implies `location` will always have a value.

Reply:
@JamesAndrewJackson13 any update on this remark?